analysis/abiotic/era5_climate_data_download_maliau.py

```diff
@@ -0,0 +1,127 @@
+#' ---
+#' title: ERA5-Land Monthly Averaged Climate Data Download Script for Maliau Basin (2010–2020)
+#'
+#' description: |
+#'   This Python script automates the download of monthly averaged ERA5-Land climate data for the Maliau Basin region for the years 2010 to 2020.
+#'   It uses the Copernicus Climate Data Store (CDS) API and argparse to support user input for selecting specific climate variables.
```

Comment:
You could refer to the original documentation here, something like: 'For details refer to https://cds.climate.copernicus.eu/how-to-api'